packages/edit-site/src/components/global-styles/font-library-modal/context.js

```diff
@@ -214,6 +214,7 @@ function FontLibraryProvider( { children } ) {
 			] );
 			refreshLibrary();
 			setIsInstalling( false );
```

Comment:
Nit: This can be moved to a `finally` block, along with the same call in the `catch` block, to avoid repeating it.

Reply:
Oh yeah! I've moved `setIsInstalling` and `setLibraryFontSelected` to a `finally` block here: https://github.com/WordPress/gutenberg/pull/57817/commits/a9140595cd9462cad4a0b6fa5f4a7aa4a2775032